### DISCUSSION TITLE
Why is rollback after x minutes when idle connection is missing from the new versions?

### DISCUSSION THREAD

**Discussion by disconnect5852** [dbeaver/dbeaver#22414] [General]
Hi!
I've been using dbeaver 22 at work since long time, and it has a feature to rollback on idle connections (but not disconnect) after set minutes pass, but i can't find it in 23 versions. Why? It was a very useful feature to prevent locking tables for long time if forgotten.

**Comment by LonwoLonwo** [comment]
Hello.

We have such setting for idle connections here:

![2024-01-04 13_07_58-Transparent Window](https://github.com/dbeaver/pro/assets/45152336/9eb5971e-fc48-4678-842f-d369950ad259)

And for idle transactions in the Preferences -> Connections -> Connection Types

**Reply by disconnect5852** [reply]
wow thanks, now i see
